### PR TITLE
@kadena/chainweb-stream-client - Changed default export at index to named export

### PR DESCRIPTION
**Pull request by Takadenoshi**
Default export didn't work with out CJS-targeted output

Also broke the example in the README

### CODE REVIEW

**Review by alber70g — APPROVED** [review]

File: common/changes/@kadena/chainweb-stream-client/chore-chainweb-stream-client-change-default-export_2023-07-19-12-19.json
Comment (alber70g) [inline common/changes/@kadena/chainweb-stream-client/chore-chainweb-stream-client-change-default-export_2023-07-19-12-19.json:1]: This one needs to be created with changesets now

**Review by alber70g — APPROVED** [review]

File: common/changes/@kadena/chainweb-stream-client/chore-chainweb-stream-client-change-default-export_2023-07-19-12-19.json
Comment (Takadenoshi) [reply common/changes/@kadena/chainweb-stream-client/chore-chainweb-stream-client-change-default-export_2023-07-19-12-19.json:1]: done

**Comment by Takadenoshi** [comment]
This needs publishing, the 0.1.0 version that was published to npm from main overwrote the 0.0.10 version that had this fix

**Review by alber70g — APPROVED** [review]

**Comment by webpro** [comment]
> This needs publishing, the 0.1.0 version that was published to npm from main overwrote the 0.0.10 version that had this fix

- We should not bump `version` ourselves manually (Changesets does this for us)
- On the same note, we do need to make sure the changelog reflects the bump

I see `minor` in the changelog, but a `patch` bump in `package.json#version`

**Comment by Takadenoshi** [comment]
- I reverted the manual bump in package.json
- I changed the changeset bump to a patch

Ideally I wanted to stay on 0.0.x for a while until it is a bit more stable but now we are on 0.1.x

**Pull request merged**